requirements.txt

```diff
@@ -2,3 +2,5 @@ cruft==2.16.0
 mex-release==0.3.0
 pdm==2.22.3
 pre-commit==4.1.0
+
+reflex==0.7.0
```

Comment:
reflex oopsie?

Reply:
yeah, super annoying. i think they try to do it foolproof, but thereby mess with everyone not following their ideas of a reflex setup.

fixed 35ad8786dae8557bd802266c1116f4d219962db5